app/javascript/dashboard/routes/dashboard/conversation/ConversationInfo.vue

```diff
@@ -1,152 +1,103 @@
-      class="conversation--attribute"
-    />
-    <contact-details-item
-      v-if="referer"
-      :title="$t('CONTACT_PANEL.INITIATED_FROM')"
-      :value="referer"
-      class="conversation--attribute"
-    >
-      <a :href="referer" rel="noopener noreferrer nofollow" target="_blank">
-        {{ referer }}
-      </a>
-    </contact-details-item>
-    <contact-details-item
-      v-if="browserName"
-      :title="$t('CONTACT_PANEL.BROWSER')"
-      :value="browserName"
-      class="conversation--attribute"
-    />
-    <contact-details-item
-      v-if="platformName"
-      :title="$t('CONTACT_PANEL.OS')"
-      :value="platformName"
-      class="conversation--attribute"
-    />
-    <contact-details-item
-      v-if="ipAddress"
-      :title="$t('CONTACT_PANEL.IP_ADDRESS')"
-      :value="ipAddress"
-      class="conversation--attribute"
-    />
-    <custom-attributes
-      attribute-type="conversation_attribute"
-      attribute-class="conversation--attribute"
-      :class="customAttributeRowClass"
-    />
-    <custom-attribute-selector attribute-type="conversation_attribute" />
-  </div>
-</template>
-
-<script>
-import { getLanguageName } from '../../../components/widgets/conversation/advancedFilterItems/languages';
+<script setup>
+import { computed } from 'vue';
+import { getLanguageName } from 'dashboard/components/widgets/conversation/advancedFilterItems/languages';
 import ContactDetailsItem from './ContactDetailsItem.vue';
 import CustomAttributes from './customAttributes/CustomAttributes.vue';
-import CustomAttributeSelector from './customAttributes/CustomAttributeSelector.vue';
-
-export default {
-  components: {
-    ContactDetailsItem,
-    CustomAttributes,
-    CustomAttributeSelector,
+const props = defineProps({
+  conversationAttributes: {
+    type: Object,
+    default: () => ({}),
   },
-  props: {
-    conversationAttributes: {
-      type: Object,
-      default: () => ({}),
-    },
-    contactAttributes: {
-      type: Object,
-      default: () => ({}),
-    },
+  contactAttributes: {
+    type: Object,
+    default: () => ({}),
   },
-  STATIC_ATTRIBUTES: [
+});
+
+const referer = computed(() => props.conversationAttributes.referer);
+const initiatedAt = computed(
+  () => props.conversationAttributes.initiated_at?.timestamp
+);
+const browserName = computed(() => {
+  if (!props.conversationAttributes.browser) {
+    return '';
+  }
+  const { browser_name: browser = '', browser_version: browserVersion = '' } =
+    props.conversationAttributes.browser;
+  return `${browser} ${browserVersion}`;
+});
+const browserLanguage = computed(() =>
+  getLanguageName(props.conversationAttributes.browser_language)
+);
+const platformName = computed(() => {
+  const { browser } = props.conversationAttributes;
+  if (!browser) {
+    return '';
+  }
+  const { platform_name: platform, platform_version: platformVersion } =
+    browser;
+  return `${platform || ''} ${platformVersion || ''}`;
+});
```

Comment:
Can we extract the `browser` and `platform` objects from before to avoid multiple accesses to nested properties.
```suggestion
const browser = props.conversationAttributes.browser || {};
const browserName = computed(() => {
  const { browser_name: browser = '', browser_version: browserVersion = '' } = browser;
  return `${browser} ${browserVersion}`;
});

const browserLanguage = computed(() =>
  getLanguageName(props.conversationAttributes.browser_language)
);

const platform = browser.platform || {};
const platformName = computed(() => {
  const { platform_name: platform = '', platform_version: platformVersion = '' } = platform;
  return `${platform} ${platformVersion}`;
});
```